src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
         {
-            var value = Get(configuration, typeof(T));
+            var value = configuration.Get(typeof(T));
```

Comment:
I suggest rename `value` => `instance`. Value is overloaded in Configuration with a different thing.